### PR TITLE
Alert & button 

### PR DESCRIPTION
**Pull request by prathameshpcodes**

### CODE REVIEW

File: pages/components/button.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                <pre class="pre-code">
+                  <code class="language-html">      
+                    &lt;button class="btn btn-error-outline">Error&lt;/button>
+                    &lt;button class="btn btn-primary-outline">Primary&lt;/button>
+                    &lt;button class="btn btn-secondary-outline">Secondary&lt;/button>
+                    &lt;button class="btn btn-success-outline">Success&lt;/button>      
+                    
+                    
+                    
+                  </code>
```
Comment (abhijeetekad) [inline pages/components/button.html:142]: Remove these blank lines.

**Review by abhijeetekad — APPROVED** [review]
Looks good to merge.

**Review by rohitrout — APPROVED** [review]
code looks good.

File: pages/components/button.html [hunk 2]
```diff
@@ -78,4 +78,150 @@ <h3 class="my-1p5">Components</h3>
...
+                    
+                </div>
+    
+                <pre class="pre-code">
+                  <code class="language-html">      
+                    &lt;button class="btn">Default&lt;/button>
+                    &lt;button class="btn btn-error-solid">Error&lt;/button>
+                    &lt;button class="btn btn-primary-solid">Primary&lt;/button>
+                    &lt;button class="btn btn-secondary-solid">Secondary&lt;/button>
+                    &lt;button class="btn btn-success-solid">Success&lt;/button>
```
Comment (kaustubh-jsr) [inline pages/components/button.html:110]: Here the classes added are of the format,btn-[type]-solid, where type could be error, primary etc. But from line 94 to 97 I can see [type]-btn-solid. You might want to have a look.

File: pages/components/button.html [hunk 3]
```diff
@@ -78,4 +78,150 @@ <h3 class="my-1p5">Components</h3>
...
+                    <button class="btn sucess-btn-outline">Sucess</button>
+                    
+                </div>
+    
+                <pre class="pre-code">
+                  <code class="language-html">      
+                    &lt;button class="btn btn-error-outline">Error&lt;/button>
+                    &lt;button class="btn btn-primary-outline">Primary&lt;/button>
+                    &lt;button class="btn btn-secondary-outline">Secondary&lt;/button>
+                    &lt;button class="btn btn-success-outline">Success&lt;/button>
```
Comment (kaustubh-jsr) [inline pages/components/button.html:138]: Similar issue here. I am not too sure if this is an issue.  btn-[type]-outline on lines 132 to 135, and [type]-btn-outline on lines 123 to 126. Pardon me if this was intended. I might be wrong here.

**Review by kaustubh-jsr — COMMENTED** [review]
The code and UI both look good to me, there are a few things I wasn't sure about, I have added them as comments. Rest is good to merge.

**Pull request merged**